src/database/queries/userChangemakerPermissions/deleteOne.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM user_changemaker_permissions
+  WHERE user_keycloak_user_id = :userKeycloakUserId
```

Comment:
This line should not be indented:

```suggestion
WHERE user_keycloak_user_id = :userKeycloakUserId
```

I know I need to dust off the SQL linter PR! :sweat_smile: